generated/nidaqmx/nidaqmx.proto

```diff
@@ -3135,7 +5018,7 @@ enum LogicFamily {
   LOGIC_FAMILY_UNSPECIFIED = 0;
   LOGIC_FAMILY_2POINT_5_V = 14620;
   LOGIC_FAMILY_3POINT_3_V = 14621;
-  LOGIC_FAMILY_5_V = 14619;
+  LOGIC_FAMILY_5V = 14619;
 }
```

Comment:
This breaks source compatibility and it's inconsistent with the other values in this enum.

Reply:
This change is not okay.

Reply:
Added a fix in https://github.com/ni/grpc-device-scrapigen/pull/127/commits/2ff22d5b025bf6287e43aff545216a1ba34eca4a